gabbi/case.py

```diff
@@ -20,16 +20,15 @@
 from collections import OrderedDict
 import copy
 import functools
+import http.cookies
 import os
 import re
 import sys
 import time
 import unittest
+import urllib. parse as urlparse
```

Comment:
minor typo?

```suggestion
import urllib.parse as urlparse
```

Reply:
Of course.

I guess the 3 changes are 2.95% OK so let's do it.

I m typing this on the phone so cannot fix it myself.